_build/composer.json

```diff
@@ -12,8 +12,8 @@
     },
     "require": {
         "php": ">=7.4",
-        "symfony/console": "^5.2",
-        "symfony/process": "^5.2",
-        "symfony-tools/docs-builder": "^0.15.0"
+        "symfony/console": "^5.4",
+        "symfony/process": "^5.4",
```

Comment:
symfony/process looks unused now

Reply:
Yes ... but let's wait and see if we finally revert to using the script directly. Thanks!